pkg/controllers/capr/bootstrap/controller.go

```diff
@@ -429,7 +444,7 @@ func (h *handler) reconcileMachinePreTerminateAnnotation(bootstrap *rkev1.RKEBoo
 		return h.ensureMachinePreTerminateAnnotationRemoved(bootstrap, machine)
 	}
 
-	// Only add the pre-terminate hook annotation if the corresponding machine and bootstrap are NOT deleting
+	// Only add the pre-drain hook annotation if the corresponding machine and bootstrap are NOT deleting
```

Comment:
Did you mean to change this comment? This code blocks still does what the original comment said.